internal/vinz/service.go

```diff
@@ -79,7 +79,7 @@ func (vs *VinzService) SaveKey(organizationID, projectIdOrAlternateId string, ke
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
+	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusCreated {
```

Comment:
Another bug I noticed here in testing. APIx returns a 201, not a 200. But I just made it more flexible.